cmd/snap/cmd_debug_timings.go

```diff
@@ -36,7 +41,39 @@ func init() {
 		"The timings command displays details about the time each task runs.",
```

Comment:
With i18n imported now, this line and one above should also be wrapped in `i18n.G(...)`.